src/fswAlgorithms/sensorInterfaces/simpleInstrumentController/simpleInstrumentController.h

```diff
@@ -32,6 +32,8 @@
 typedef struct {
     /* User configurable variables */
     double attErrTolerance; //!< Normalized MRP attitude error tolerance
+    unsigned int useRateTolerance; //!< Flag to enable rate error tolerate
+    double rateErrTolerance; //!< Rate error tolerance in rad/s
```

Comment:
In the 2nd commit you are reversing some things from the first commit.  Please squash these two commits into a single commit.  You saw the Git workshop Patrick did in December? We recorded the video of that on how to do such branch cleanup operations.  

I would prefer the documentation commit to be done separate from the code.  You should be able to split that out of this combined commit into a separate commit.

Reply:
:small_red_triangle_down: Seconded. Happy to assist if needed.

Reply:
Cleaned commit structure to have 1 for code and 1 for documentation.